src/status_im/contexts/wallet/collectible/view.cljs

```diff
@@ -28,15 +28,16 @@
      collection-name]]])
 
 (defn cta-buttons
-  [chain-id token-id contract-address]
+  [chain-id token-id contract-address watch-only?]
```

Comment:
nit, probably better to make this a map now

Reply:
Sure 👍